executor/cte.go

```diff
@@ -760,8 +759,8 @@ func (p *cteProducer) checkAndUpdateCorColHashCode() bool {
 	return changed
 }
 
-func (p *cteProducer) logTbls(ctx context.Context, err error, iterNum uint64) {
```

Comment:
irrelevant modification?

Reply:
info log when failpoint fail to help debug problems